pom.xml

```diff
@@ -751,6 +751,12 @@
                 <artifactId>gson</artifactId>
                 <version>2.8.9</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <!-- using 31.0.1-android flavor instead of 31.0.1-jre, so that we are compatible with java7 -->
+                <version>31.0.1-android</version>
```

Comment:
Maybe we can use a newer version (don't need to be compatible with java7 anymore)

Reply:
Actually, we already used 32.x.x a couple of lines below - this old version is actually a merge error.
Updated to 33.0.0-jre now.